### PR TITLE
Most of the requested items from "could you add a few options #255"

### PR DESCRIPTION
**Pull request by adambennett55**
Only haven't added the event to trigger when vbm is controlling movement.

### CODE REVIEW

File: BossMod/Framework/Plugin.cs [hunk 1]
```diff
@@ -46,8 +47,8 @@ public sealed class Plugin : IDalamudPlugin
         private WorldStateGame _ws;
         private BossModuleManager _bossmod;
         private Autorotation _autorotation;
-        private AI.AIManager _ai;
-        private AI.Broadcast _broadcast;
+        public static AIManager _ai;
```
Comment (awgil) [inline BossMod/Framework/Plugin.cs:50]: Please no :(

File: BossMod/Framework/Plugin.cs [hunk 2]
```diff
@@ -161,6 +162,38 @@ private void OnCommand(string cmd, string args)
                 case "r":
                     _wndReplay.SetVisible(!_wndReplay.IsOpen);
                     break;
+                case "aion":
```
Comment (awgil) [inline BossMod/Framework/Plugin.cs:165]: Why is that needed, considering there's already a 'cfg' command for modifying arbitrary options?

File: BossMod/AI/AIBehaviour.cs [hunk 3]
```diff
@@ -36,7 +36,8 @@ public void Execute(Actor player, Actor master)
                 return;
 
             // keep master in focus
-            FocusMaster(master);
+            if (Plugin._ai._config.FocusTargetLeader)
```
Comment (awgil) [inline BossMod/AI/AIBehaviour.cs:39]: Config is accessible directly via Service.Config.Get if needed, was this the only reason for making the _ai field static?

File: BossMod/AI/AIManager.cs [hunk 4]
```diff
@@ -7,11 +7,11 @@
 
 namespace BossMod.AI
 {
-    class AIManager : IDisposable
+    public class AIManager : IDisposable
     {
-        private Autorotation _autorot;
+        public Autorotation _autorot;
```
Comment (awgil) [inline BossMod/AI/AIManager.cs:12]: The plugin follows standard c# naming conventions (private fields _likeThis, public LikeThis). See https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

**Comment by awgil** [comment]
> Sorry I'll fix it

Btw you don't have to close/recreate PR to update it :) just push changes and it will be updated automatically!

**Comment by adambennett55** [comment]
It's been updated sorry for the trouble
